benchmarks/release_benchmarking/README.md

```diff
@@ -0,0 +1,107 @@
+# Release Benchmarking Guide
```

Comment:
Can we describe the types of benchmarks we are running?

Reply:
Yes, they are described in the [results template file](https://github.com/nvidia-holoscan/holohub/pull/472/files#diff-e0e1641209082ebe2d81305e6095bbb992cb095c2ee4b6129c21da1587fcba58), I'll add a short overview and a reference to the README.

Reply:
Updated the README